craiglister.py

```diff
@@ -232,16 +306,37 @@ def getOrderedListingImages(listingFolder):
 listingFolders = [listing for listing in os.listdir(listingsFolderDirectory) if listing[0] != "." and listing != "listed"]
 
 for listingFolder in listingFolders:
+
+    #This is the conditional argument for logins
+    if cycleNum <= 2:
+        gmailUser = os.environ.get("GMAIL1")
+        gmailPass = os.environ.get("GMAIL_PASS1")
+        print("Used first email: ", cycleNum)
+
+    else:
+        gmailUser = os.environ.get("GMAIL2")
+        gmailPass = os.environ.get("GMAIL_PASS2")
+        print("Used second email: ", cycleNum)
+
+
     listingFolder = os.path.abspath(os.path.join(listingsFolderDirectory, listingFolder))
     with open(os.path.abspath(os.path.join(listingFolder, 'info.txt')), 'r') as info:
         listing = listingInfoParse(info.read())
     listing.images = getOrderedListingImages(listingFolder)
     listing.driver = webdriver.Chrome(chromedriver)
```

Comment:
So the Chromedriver in this branch is the ChromeDriver.exe which works in windows, but we're going to need the Mac Chromedriver in the folder fo me to use it and we're going to need the Linux driver for it to work on AWS. Just FYI